package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mediarithmics/plugins-nodejs-sdk",
-  "version": "0.7.10",
+  "version": "0.7.11",
```

Comment:
https://docs.google.com/document/d/1BKV_oqmIFDwWZp3w-Ah3uz66keDYF0x0MNeHQn3lh1E/edit

it's not up to you to bump this